Taskfile.yml

```diff
@@ -56,6 +57,10 @@ tasks:
         sh: task drush -- user:login --no-browser
     silent: true
 
+  build-assets:
```

Comment:
I would call this task `assets-build` to namespace it (it's related to “assets”).